src/bidiMapper/domains/log/logManager.ts

```diff
@@ -68,7 +68,7 @@ export class LogManager {
         });
         const argsPromise: Promise<CommonDataTypes.RemoteValue[]> =
           realm === undefined
-            ? Promise.resolve(params.args)
+            ? Promise.resolve(params.args as CommonDataTypes.RemoteValue[])
```

Comment:
@sadym-chromium could you advice? it looks like params.args where of the wrong type

Reply:
Well, without realm we cannot do much, so just put to args whatever we got from CDP. So yes, this line should work as a backup.